app/forms/personal_details_form.rb

```diff
@@ -131,4 +141,23 @@ def has_valid_nino?
     valid?
     errors.exclude?(:national_insurance_number)
   end
+
+  # FIXME RL: Remove reseting claim attributes once we've migrated student loan
+  # data to the answers model
+  def reset_depenent_answers_attributes
+    journey_session.answers.assign_attributes(
+      has_student_loan: nil,
+      student_loan_plan: nil
+    )
+
+    claim.assign_attributes(has_student_loan: nil, student_loan_plan: nil)
+
+    if journey == Journeys::TeacherStudentLoanReimbursement
+      journey_session.answers.assign_attributes(
+        student_loan_repayment_amount: nil
+      )
+
+      claim.eligibility.assign_attributes(student_loan_repayment_amount: nil)
+    end
```

Comment:
I'm going to leave this for the minute as we're going to be changing this implementation once we get round to moving the student loans data to the session answers (+ I want to avoid some rebasing pain 😂 )
Also I think being able to do `journey.student_loan?` is much better than checking the module here so we should def add a query method to the journey to support that.